src/main/groovy/life/qbic/samplestatus/reporter/api/UserDetails.groovy

```diff
@@ -0,0 +1,25 @@
+package life.qbic.samplestatus.reporter.api
+
+import groovy.transform.ToString
+
+/**
+ * <p>A users object that represents a users responsible for a location</p>
+ *
+ * @since <version tag>
+ */
+@ToString
+class UserDetails {
+    String firstName
+    String lastName
+    String email
+
+    UserDetails(String firstName, String lastName, String email) {
+        this.firstName = firstName
+        this.lastName = lastName
+        this.email = email
+    }
+
+    String getFullName() {
+        return getFirstName() + " " + getLastName()
+    }
```

Comment:
We should go for JPA entities here, since we retrieve data from our persistence layer and can avoid boiler plate code of mapping pure data back and forth

Reply:
The thing is that the database for this data is currently convoluted with a lot of other fields. I do not know how to handle this case. To make it work, I implemented it this way first.